packages/cli/src/local-data/replay-assets.ts

```diff
@@ -84,3 +101,36 @@ export const fetchAsset: (path: string) => Promise<string> = async (path) => {
   await saveAssetMetadata(assetMetadata);
   return filePath;
 };
+
+const getOrCreateAssetsDir: () => Promise<string> = async () => {
```

Comment:
No change to these functions. They've just been moved to the bottom, as per step-down rule, and the 'export' prefix removed (they're only used locally in this file).